main.go

```diff
@@ -50,11 +52,41 @@ var tsclient *tailscale.LocalClient
 func main() {
 	flag.Parse()
 
-	log.Println("Verifying we can fetch machines from Mosyle...")
-	_, err := enumerateMachines()
+	log.Println("Logging in to Mosyle ...")
+	err := mosyleLogin()
+	if err != nil {
+		log.Fatalf("failed to login to mosyle: %v", err)
+	}
+
+	go func() {
+	DailyRefresh:
+		for {
+			time.Sleep(23 * time.Hour)
+
+			err := mosyleLogin()
+			if err == nil {
+				continue DailyRefresh
+			}
+
+			log.Printf("failed to login to mosyle, trying again several more times: %v\n", err)
+			for i := 0; i <= 55; i += 1 {
+				time.Sleep(55 * time.Second)
```

Comment:
Why 55?

Reply:
each attempt takes ~ a while, so I figured I'd just make it a little less than a full minute. I dunno :)

Reply:
Let's just make it a full minute so that it's less "magical" seeming